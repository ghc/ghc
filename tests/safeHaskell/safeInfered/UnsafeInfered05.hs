{-# LANGUAGE Haskell2010 #-}
{-# LANGUAGE Safe #-}
module UnsafeInfered05 where

import safe UnsafeInfered05_A

g :: Int
g = f

