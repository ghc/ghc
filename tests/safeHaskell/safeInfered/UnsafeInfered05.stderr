
UnsafeInfered05.hs:5:1: error: [GHC-44360]
    UnsafeInfered05_A: Can't be safely imported!
    The module itself isn't safe.
